api/nakadi-event-bus-api.yaml

```diff
@@ -871,6 +871,35 @@ paths:
           description: Unprocessable Entity
           schema:
             $ref: '#/definitions/Problem'
+    patch:
+      tags:
+        - subscription-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        Endpoint for reseting subscription committed offsets to provided.
+      parameters:
+        - name: subscription_id
+          in: path
+          type: string
+          description: Id of subscription
+          required: true
+        - name: cursors
+          in: body
+          schema:
+            type: object
+            properties:
+              items:
+                description: |
+                  List of cursors that the consumer acknowledges to have successfully processed.
+                type: array
+                items:
+                  $ref: '#/definitions/SubscriptionCursor'
+            required:
+              - items
+      responses:
+        '200':
+          description: Offsets were reseted
```

Comment:
I also think that it makes sense to describe these two cases in api (subscription not found and incomplete list of cursors) - because here it is part of behavior of our endpoint.